### DISCUSSION TITLE
Custom role in grid architecture

### DISCUSSION THREAD

**Discussion by jedansi** [Security-Onion-Solutions/securityonion#7149] [Unsupported Versions] [answered]
Hello
I realized a distributed grid architecture, how I can create a user role for each node in which the user only sees the information (ALERT, kibana information) relating to his own node.
 thank you

**Comment by presianbg** [comment, marked as answer]
Hi, 

Haven't tried it yet, but have you looked at this example from the docs ?
https://docs.securityonion.net/en/latest/rbac.html?highlight=custom%20roles#defining-security-onion-roles
